library/ssl_tls12_client.c

```diff
@@ -1972,7 +1972,7 @@ static int ssl_parse_server_psk_hint( mbedtls_ssl_context *ssl,
 
     /*
      * Note: we currently ignore the PKS identity hint, as we only allow one
```

Comment:
```suggestion
     * Note: we currently ignore the PSK identity hint, as we only allow one
```

Reply:
Nice spot - thanks - updated here and in the backport (sorry @davidhorstmann-arm )